packages/react-examples/.storybook/preview.js

```diff
@@ -5,68 +5,69 @@ import 'cypress-storybook/react';
 import { withPerformance } from 'storybook-addon-performance';
 import { withKeytipLayer, withStrictMode } from '@fluentui/storybook';
 
-/**
- * This placeholder will be replaced with the actual package name by custom webpack loader `./preview-loader.js`
- * @type {string}
- */
-const packageNamePlaceholder = 'PACKAGE_NAME';
-
 initializeIcons();
 
+// This API is deprecated (in favor of `export const decorators = []`) but the new way appears not
+// to work when using the legacy configure() API
 addDecorator(withPerformance);
 addDecorator(withStrictMode);
 addDecorator(withKeytipLayer);
 
+// This API is deprecated (in favor of `export const parameters = {}`) but same issue as above
 addParameters({
-  a11y: {
+  a11y: /** @type {import('@storybook/addon-a11y').A11yParameters} */ ({
     manual: true,
-  },
+  }),
 });
 
+// This API is deprecated and will no longer work in storybook 7 or with storyStoreV7.
+// https://github.com/storybookjs/storybook/blob/next/MIGRATION.md#deprecated-configure
+// The selective loading functionality currently implemented by preview-loader can be more easily
+// handled by just settings `stories: [/*glob*/]` in main.js, but it's unclear if a replacement is
+// available for transforming files before loading as stories.
 configure(loadStories, module);
 
-export const parameters = {};
-
-// ================================
-//          Helpers
-// ================================
+// ==========================================================
+//   Helpers for loading examples in component story format
+// ==========================================================
 
 /**
  * @typedef {{
  *   default: { title: string };
- *   [subStoryName: string]: React.FunctionComponent | { title: string };
```

Comment:
Lots of naming and doc improvements here, now that I have a better understanding of what this is doing and the reason for the output format

Reply:
thanks for adding those. In general I'd rather have test then lot of comments that can became obsolete quick.